src/app/model/staypoints.ts

```diff
@@ -39,3 +46,36 @@ export function writeStaypointsToGeoJSON(sp: StayPoints, path: string) {
     }
   })
 }
+
+export function writeStaypointClusterArrayToGeoJSON(
+  spcs: StayPointCluster[],
+  path: string
+) {
+  const geojson = {
+    name: 'StayPointClusters',
+    type: 'FeatureCollection',
+    features: [],
+    properties: {
+      trajID: spcs[0].trajID,
```

Comment:
Make sure that `spcs` is not empty before accessing it.

Reply:
Makes sense, I changed it.